datafusion/functions-aggregate/src/first_last.rs

```diff
@@ -72,7 +72,7 @@ impl Default for FirstValue {
 impl FirstValue {
     pub fn new() -> Self {
         Self {
-            aliases: vec![String::from("first_value")],
+            aliases: vec![],
```

Comment:
A minor comment is we can probably remove the `aliases` field entirely and just return an empty slice

```rust
    fn aliases(&self) -> &[String] {
        &[]
    }
```